lib/Listener.php

```diff
@@ -106,12 +106,16 @@ public function logoutUser(): void {
 		/** @var string[] $sessionIds */
 		$sessionIds = $this->talkSession->getAllActiveSessions();
 		foreach ($sessionIds as $sessionId) {
-			$room = $this->manager->getRoomForSession($user->getUID(), $sessionId);
-			$participant = $room->getParticipant($user->getUID());
-			if ($participant->getInCallFlags() !== Participant::FLAG_DISCONNECTED) {
-				$room->changeInCall($participant, Participant::FLAG_DISCONNECTED);
+			try {
+				$room = $this->manager->getRoomForSession($user->getUID(), $sessionId);
+				$participant = $room->getParticipant($user->getUID());
+				if ($participant->getInCallFlags() !== Participant::FLAG_DISCONNECTED) {
+					$room->changeInCall($participant, Participant::FLAG_DISCONNECTED);
```

Comment:
missing import(s)

Reply:
Thanks for the hint! It should be ready now.